firmware/src/temperature.c

```diff
@@ -28,6 +28,8 @@
 #include "app_settings.h"
 
 static const uint16_t SAMPLING_PERIOD = 10000;  // 1s
+// Recalibrate every 5 minutes
```

Comment:
Can you expand the comment to clarify it's 300 * SAMPLING_PERIOD, which happens to be in seconds.